src/FilesManagement.cpp

```diff
@@ -139,12 +139,23 @@ void FilesManagement::setWsClient(WSClient *c)
     {
         loadFilesCacheModel();
     });
+
+    setFileCacheControlsVisible(wsClient->get_fwVersion() >= "v1.2");
```

Comment:
This is not good. String compared like that is likely to fail. The mini version contains also `_mini` at the end.

Please implement wsClient->isFw12() like in MPDevice.

See the implementation in the backend:
https://github.com/mooltipass/moolticute/blob/02a087d2d5b16bb47ead47f35af239f71ef6995a/src/MPDevice.cpp#L400
and 
https://github.com/mooltipass/moolticute/blob/e8b7dd5894f80cff0d1c44689454f877214f4c66/src/MPDevice.h#L211